plugins/Fable.React.Plugins.fs

```diff
@@ -110,8 +113,7 @@ type ReactComponentAttribute(exportDefault: bool) =
                     let getterKind = Fable.ByKey(Fable.ExprKey(AstUtil.makeStrConst arg.DisplayName))
                     let getter = Fable.Get(Fable.IdentExpr propsArg, getterKind, Fable.Any, None)
                     (arg, getter)::bindings)
-                |> List.rev
-                |> fun bindings -> Fable.Let(bindings, decl.Body)
+                |> List.fold (fun body (k,v) -> Fable.Let(k, v, body)) decl.Body
```

Comment:
At the end, I've separated `Expr.Let` and `Expr.LetRec` as in F# AST so if you have a list of bindings you need to fold them (in reverted order).